packages/compass-e2e-tests/tests/instance-databases-tab.test.ts

```diff
@@ -137,32 +137,38 @@ describe('Instance databases tab', function () {
     await browser.waitUntilActiveInstanceTab('Databases');
   });
 
-  it('can refresh the list of databases using refresh controls', async function () {
-    const db = 'my-instance-database';
-    const coll = 'my-collection';
+  it.only('can refresh the list of databases using refresh controls', async function () {
+    const db = 'test'; // added by beforeEach
     const dbSelector = Selectors.databaseCard(db);
 
-    // Create the database and refresh
+    // Browse to the databases tab
+    await browser.navigateToInstanceTab('Databases');
+
+    // Make sure the db card we're going to drop is in there.
+    await browser.scrollToVirtualItem(
+      Selectors.DatabasesTable,
+      dbSelector,
+      'grid'
+    );
+    await browser.$(dbSelector).waitForDisplayed();
+
+    // Wait for the page to finish loading as best as we can
+    await browser.waitUntil(async () => {
+      const placeholders = await browser.$$(Selectors.DatabaseStatLoader);
+      return placeholders.length === 0;
```

Comment:
If only we (or a user) could reliably tell in the UI once everything is done loading. 🤔